### ISSUE TITLE
Fund social accounts

### ISSUE THREAD

**Issue by brandonhaslegs** [drips-network/app#808]
The most important ones I can think of are:

- [ ] GitHub accounts/orgs
- [ ] X handles
- [ ] Instagram/Threads handles

**Comment by efstajas** [comment]
Let's track adding individual external identity anchors when we have specific reasons to do so.

**Comment by brandonhaslegs** [comment]
I don't think we need a specific reason to do this. We should add these proactively. At least GitHub and X handles. Don't care about Instagram. It would be a GREAT feature that would remove A LOT of overhead to just fund someone's Twitter or GitHub account.
